lib/Client.php

```diff
@@ -49,7 +49,7 @@ public function getDefaultTimeout()
 
     /**
      * @param RequestInterface $apiRequest
-     * @return \stdClass
+     * @return mixed
```

Comment:
The difference here is that this method strictly returns an `stdClass`, but other methods that consumes it may have their own return types as they wish. In order to keep the strict reference, I believe that keeping the `stdClass` reference is the best approach.

@williamokano what do you think?